english/Raiders/LetsgoRaiders.md

```diff
@@ -3,5 +3,5 @@ Lets go Raiders!
 [Here's the link](https://www.google.com)
 
 You see Batman at the Raiders game you are attending and you decide
-to go ask him some personal questions. [This is what you find](
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.